bridges/scalajs-1/src/main/scala/bloop/scalajs/JsBridge.scala

```diff
@@ -85,7 +85,7 @@ object JsBridge {
       (config.kind, scalaJSModuleKindSplitStyle) match {
         case (ModuleKindJS.ESModule, Some(value)) =>
           StandardImpl.clearableLinker(
-            linkerConfig.withModuleSplitStyle(value)
+            linkerConfig.withModuleSplitStyle(value).withMinify(isFullLinkJS)
```

Comment:
In order to be consistent with sbt-scalajs, you should enable `minify` for *all* `fullLink` builds, regardless of `ModuleKind`.

Even when used in conjunction with the Closure compiler, `minify` still brings some code size improvements: the optimizer works extra hard to reduce the need for temporary local variables.